source/features/reactions-avatars.css

```diff
@@ -4,8 +4,8 @@
 }
 
 button.reaction-summary-item { /* `button` excludes the "Add reaction" icon */
-	border-top: 1px solid var(--color-border-primary, #e1e4e8); /* Required when the second line is longer than the first line */
-	border-bottom: 1px solid var(--color-border-primary, #e1e4e8); /* Required when the first line is longer than the second line */
+	border-top: 1px solid var(--github-border-color); /* Required when the second line is longer than the first line */
```

Comment:
Why are we losing the fallbacks here?

Reply:
They’re not part of the variable itself

Reply:
So this old code was invalid `var(--color-border-primary, #e1e4e8)`? Man, CSS has somehow gotten itself even more complicated.

Reply:
> So this old code was invalid

No, it's just that now the fallback is defined directly on `--github-border-color`. DRY and all that.

Reply:
Are we defining that? As enterprise appears to have zero variables when I checked the other day.

Reply:
Yes, see the `refined-github.css` file diff.

Reply:
Thanks, repo has been moving very fast lately and it's hard to keep up.